Source/Core/Matrix4.js

```diff
@@ -675,9 +643,7 @@ define([
         if (far <= 0.0) {
```

Comment:
Right, thats why I left it as is until we can figure out a more robust solution.